Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rpm"
-version = "0.13.0"
+version = "0.13.1"
```

Comment:
If the above breaks anything, we should make sure to also do a minor crate version bump.

Reply:
I should clarify, 0.13.1 is already released but the version hasn't been bumped on main branch yet - I forgot to do that when merging the changelog, so I ended up just adjusting that locally when I did the release.

Prior to the next release, it would be bumped again alongside the changelog.